app/src/main/java/org/wikipedia/talk/TalkTopicActivity.kt

```diff
@@ -125,6 +127,22 @@ class TalkTopicActivity : BaseActivity(), LinkPreviewDialog.Callback {
         onInitialLoad()
     }
 
+    override fun onCreateOptionsMenu(menu: Menu): Boolean {
+        menuInflater.inflate(R.menu.menu_talk_topic, menu)
+        return true
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        super.onOptionsItemSelected(item)
+        return when (item.itemId) {
+            R.id.menu_talk_topic_share -> {
+                ShareUtil.shareText(this, getString(R.string.talk_share_discussion_subject, topic?.html?.ifEmpty { getString(R.string.talk_no_subject) }), pageTitle.uri + "#" + topic?.html)
```

Comment:
If the section title contains spaces (e.g. "Name of the house"), then it will not be redirected to the section correctly.